plugins/tableselection/plugin.js

```diff
@@ -692,7 +692,7 @@
 
 			// Pasted value must be filtered using dataProcessor to strip all unsafe code
 			// before inserting it into temporary container.
-			tmpContainer.setHtml( dataProcessor.toHtml( dataValue ), {
+			tmpContainer.setHtml( CKEDITOR.tools.htmlSafeByReview(dataProcessor.toHtml( dataValue ), 'dataValue goes through filtration to remove unsafe code'), {
```

Comment:
So the crux of this is whether core/htmldataprocessor.js is a sufficient enough HTML sanitizer or not, right?

Are there enough instances of data processors calling `.toHtml` where the output is then modified before being inserted into the page? If not, should `.toHtml` do the blessing to TT itself? (Or create a newly-named version of `.toHtml`, like `.toTrustedHtml` that outputs an already-converted type if we're scared about breaking existing instances of `.toHtml`.)

Reply:
I think it might be better if we legacy convert dataprocessor.toHtml. The function strips the content of textareas and disables some scripts but I'm not entirely sure if that covers all possible XSS attacks. In that case would it be better still to create another function like .toHtmlLegacy or do we keep .toHtml but wrap in legacy conversions where needed?

Reply:
That's a matter of aesthetics! If we want to create a `.toHtmlLegacy` that just calls `.toHtml` plus a legacy conversion immediately, why not?

Reply:
Sounds good, moving legacy conversion to PR # 11